regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -93,6 +93,11 @@ var PreambleView = ChildView.extend({
     this.mode = 'write';
     $parent = $parent.clone();
     $parent.find('.activate-write').remove();
+
+    if (section.indexOf('cfr') > -1) {
+      label = '§ ' + label;
+    }
```

Comment:
You're right, after I updated marker setting in `human_label`, the labels in HTML data updated accordingly so I don't need that anymore. Thanks. 👍